index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css" /> 
+    <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="background"><img src="./headline-mobile.png"></div>
+    <div class="top">
+        <div class="header">
+            <p class="logo">My Logo</p>
+        </div>
+        <div class="menuhover">
+            <button>
+                 <i class="fa-solid fa-bars"></i>
+            </button>
+        </div>
```

Comment:
- [ ] [OPTIONAL] This hamburger button should be wrapped with a `nav` tag.  * The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.

**_I am marking this comment as optional because the previous reviewer did not mention it. But I strongly suggest you fix these issues as they are core requirements that the previous reviewer missed._**

Reply:
thanks for advices @bobb-Rob